tests/test_crawlera.py

```diff
@@ -102,7 +102,11 @@ def _assert_enabled(self, spider,
         # check for not banning before maxbans for bancode
         for x in range(maxbans + 1):
             self.assertEqual(crawler.engine.fake_spider_closed_result, None)
-            res = Response('http://ban.me/%d' % x, status=self.bancode)
+            res = Response(
+                'http://ban.me/%d' % x,
+                status=self.bancode,
+                headers={'X-Crawlera-Error': 'banned'}
```

Comment:
please add a trailing command to the end of this line and please rebase your PR with master so the build can pass. Thanks